exec/cnex/cnex.c

```diff
@@ -1388,8 +1418,15 @@ void exec_ALLOC(TExecutor *self)
     self->ip++;
     uint32_t val = exec_getOperand(self);
 
-    Cell *cell = cell_createArrayCell(val);
+    Cell *cell = heap_allocObject(self);
+    cell->array = array_createArrayFromSize(val);
+    cell->type = cArray;
     push(self->stack, cell_fromAddress(cell));
+    self->allocations++;
```

Comment:
Where does `allocations` get reset to zero?

Reply:
Never.  That's the allocation count.  When this supports an actual collector, then it will reset when it sweeps.